packages/elements/src/components/ino-markdown-editor/ino-markdown-editor.tsx

```diff
@@ -84,30 +77,50 @@ export class MarkdownEditor implements ComponentInterface {
     this.editor = new Editor({
       element: this.editorRef,
       extensions: [StarterKit, Link],
-      onBlur: () => this.valueChange.emit(this.htmlToMarkdown()),
+      onBlur: () => {
+        const markdownText = this.htmlToMarkdown();
+        if (this.isMarkdownValid)
+          this.valueChange.emit(markdownText);
+      },
       onTransaction: () => (this.stateChanged = !this.stateChanged),
     });
   }
 
   private onTextareaChange = (e: CustomEvent<string>) => {
     e.stopPropagation();
-    this.editor.commands.setContent(this.markdownToHtml(e.detail));
     this.textareaRef.value = e.detail;
   };
 
   private onTextareaBlur = (e: CustomEvent<void>) => {
```

Comment:
If you you are in Markdown-Mode, the `tiptap`-Editor `blur`-Event will not triggered. Beside this the "validation" of the text is slight different compared to the Editor. Here you can see that I deleted line 94. I do this because of possible invalid intermediate states by typing raw markdown in textarea. To validate the input I shift this to the `blur`-Event of the textarea.

Reply:
Okay, so there is basically a validation method which is called while blurring in preview mode and another one while blurring in markdown mode? Can we name the methods accordingly then?
Like `handlePreviewBlur` and `handleMarkdownBlur`?